sentinel-adapter/sentinel-apache-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/BaseSentinelDubboFilter.java

```diff
@@ -35,47 +38,69 @@ public BaseSentinelDubboFilter() {
         this.listener = new SentinelDubboListener();
     }
 
-    static class SentinelDubboListener implements Listener {
+
```

Comment:
It's better to keep only one blank line here.
Also Line#78,Line#80.
Line#104 can be removed for better and uniform formatting.